cmd/nginx-ingress/main.go

```diff
@@ -247,6 +250,11 @@ func main() {
 		glog.Fatal("enable-tls-passthrough flag requires -enable-custom-resources")
 	}
 
+	if *enablePreviewPolicies {
+		glog.Warning("enable-preview-policies is deprecated. Please use -enable-oidc instead.")
```

Comment:
enable-preview-policies is deprecated. To enable OIDC Policies please use -enable-oidc instead.

Making the statement more clear.

Reply:
Done